scripts/upload-releasenotes.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+BRANCH="$(git symbolic-ref --short HEAD)"
+NOTES=${1:-release-notes.txt}
+
+case "$BRANCH" in
+	v[1-9]*)
+		;;
+	master)
+		echo "this is master!"
+		exit 1
+		;;
+	*)
+		echo "this is not a release branch!"
+		exit 1
+		;;
+esac
+
+if [ ! -f $NOTES ]; then
+	echo "cannot find release notes!"
+	echo "Please point to correct location:"
+	echo "$0 <file>"
+	exit 1
+fi
+
+PLATFORMS="$(grep '#MACHINE ?=' conf/local.conf.sample | awk '{ print $3 }' | tr -d '"')"
+
+for platform in $PLATFORMS; do
+	aws s3 cp $NOTES s3://repo.bisdn.de/pub/onie/$platform/onie-bisdn-$platform-$BRANCH-releasenotes.txt
```

Comment:
```suggestion
	aws s3 cp $NOTES s3://repo.bisdn.de/pub/onie/$platform/onie-bisdn-$platform-$BRANCH-$NOTES
```

Reply:
This one should stay a hardcoded releasenotes.txt, as this is what we have always used. The source filename should not have any influence on the target filename.